### ISSUE TITLE
I cannot find the calculating or estimating of wasserstein distance!

### ISSUE THREAD

**Issue by deahhh** [martinarjovsky/WassersteinGAN#81]

**Comment by deahhh** [comment]
In the paper, wasserstein distance is estimated with a new network f by minimum the minus of the two distribution, which is not displayed in this repository.

**Comment by milleniums** [comment]
> In the paper, wasserstein distance is estimated with a new network f by minimum the minus of the two distribution, which is not displayed in this repository.

I don't understand your explanation, meaning that the code doesn't implement EMD?

**Comment by milleniums** [comment]
I can see what you mean by reading this [blog](https://zhuanlan.zhihu.com/p/539156191).
